### PR TITLE
BUGFIX: Fix some issues with the history module

### PR DESCRIPTION
**Pull request by kitsunet**
This fixes the following issue with the history module:

- The "load more" button label is now translateable
- The NodeEvent won't call the SiteRepository if there is no "site"
- The EventRepository allows to filter correctly by Events
- HistoryController will correctly paginate due to the fixed Repository

NEOS-1221 #comment some cleanup to the implementation

### CODE REVIEW

**Comment by kitsunet** [comment]
@aertmann @patrickreck 
I needed to fix some more stuff to make it work right...

File: TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/HistoryController.php [hunk 1]
```diff
@@ -13,12 +13,16 @@
 
 use TYPO3\Flow\Annotations as Flow;
 use TYPO3\Flow\Mvc\View\ViewInterface;
+use TYPO3\Flow\Reflection\ObjectAccess;
```
Comment (aertmann) [inline TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/HistoryController.php]: not used

**Comment by aertmann** [comment]
:+1: makes sense and works.. had the site issue as well

**Comment by kdambekalns** [comment]
What is "the site issue"?

File: TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/HistoryController.php [hunk 2]
```diff
@@ -40,9 +44,10 @@ class HistoryController extends AbstractModuleController
      */
     public function indexAction($offset = 0, $limit = 10)
     {
-        $events = $this->eventRepository->findRelevantEvents($offset, $limit + 1)->toArray();
+        $events = $this->eventRepository->findRelevantEventsByWorkspace($offset, $limit + 1, 'live')->toArray();
```
Comment (kdambekalns) [inline TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/HistoryController.php]: Good one, when was that added? That filtering further down existed for ages already…

**Comment by kdambekalns** [comment]
Looks good. Thanks!

**Pull request merged**